docs/user/welcome/upgrade.rst

```diff
@@ -29,15 +31,26 @@ But first to upgrade - change your dependency to |release| (or an appropriate st
 GeoTools 23.x
 -------------
 
-Change to repo.osgeo.org Repository
```

Comment:
Small nitpick - this is under GeoTools 23.x, but will also affect 22.x

Reply:
Yeah not sure how to address that, ... I could start from the earlier version affected.

Reply:
Don't think it matters to much either way, given that 22.x is EOL pretty soon

Reply:
That worked, considering this resolved.